webapp/channels/src/components/user_settings/display/user_settings_theme/user_settings_theme.tsx

```diff
@@ -147,18 +146,6 @@ export default class ThemeSetting extends React.PureComponent<Props, State> {
         this.props.setRequireConfirm?.(false);
     };
 
-    handleImportModal = (): void => {
-        this.props.actions.openModal({
-            modalId: ModalIdentifiers.IMPORT_THEME_MODAL,
```

Comment:
please remove this constant `IMPORT_THEME_MODAL`

Reply:
Removed.